package.json

```diff
@@ -10,6 +10,7 @@
     "revision": "5cd1dc12f984"
   },
   "scripts": {
+    "clobber": "rm -r .electron; rm build-config.json; rm -r node_modules",
```

Comment:
Nit: use something like [**rimraf**](https://www.npmjs.com/package/rimraf) for cross platform greatness.